doc/manpages/mapping.asciidoc

```diff
@@ -0,0 +1,45 @@
+KAKOUNE(1)
+==========
+
+NAME
+----
+mapping - a
+
+Description
+-----------
+Creating and removing shortcuts boils down to the following functions,
+respectively:
+
+---------------------------------------
+map [flags] <scope> <mode> <key> <keys>
+unmap <scope> <mode> <key> [<expected>]
+---------------------------------------
+
+The *map* command makes *key* behave as if the *keys* sequence was typed.
+
+*mode* dictates in what context the mapping will be available:
+
+	*insert*::
+		insert mode
+	*normal*::
+		normal, commands mode
+	*prompt*::
+		prompts, inputs
+	*menu*::
+		multiple selection menus
+	*user*::
+		user mode entered when the prefix (default: *,*) is hit
+
+Note that despite the *mode* in which the bindings are initialized, the
+keys are always executed in a 'normal' execution context.
```

Comment:
It's weird, that line came from `keys.asciidoc` and even then it wasn't the case IIRC.